components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -806,6 +818,15 @@ def leave_none_unset_int(s):
          ("If True, cors_origin_whitelist will not be used and all origins "
           "will be authorized to make cross-site HTTP requests.")],
 
+    "omero.web.timezone":
+        ["TIME_ZONE",
+         "UTC",
+         json_or_plain_str,
+         ("Local time zone for this installation. Choices can be found here: "
+          "http://www.postgresql.org/docs/8.1/static/datetime-keywords.html"
```

Comment:
Curious version of PostgreSQL to choose?

Reply:
It's copied and pasted from the old comment, and I couldn't find the equivalent page for PG 10.

Reply:
Interesting. Maybe it's not there for a reason. (Gotten very large or something.) Perhaps we should have people look at the `pg_timezone_names` instead. Needn't block this PR though.

Reply:
It turns out it's a really old link... 😂 https://github.com/openmicroscopy/openmicroscopy/pull/2585#discussion_r13531421

Reply:
I've gone for wikipedia instead

Reply:
Heh, thank you for finding a fix.